app/components/product/view.hbs

```diff
@@ -21,9 +21,26 @@
               </div>
             {{/if}}
           </div>
+        </div>
+        <div class="mt-6 grid grid-cols-1 gap-y-2 gap-x-4 md:grid-cols-6 items-center">
           <div class="col-span-6">
             <Util::Property @label="Naam" @value={{@model.name}} />
           </div>
+          {{#if @model.alternateNames.length}}
+            <div class="col-span-6 text-sm text-gray-900 italic">
```

Comment:
I would wrap them in another `<Util::Property @label="Aliases">` component. It accepts a block as well.